boxsdk/object/item.py

```diff
@@ -111,6 +110,10 @@ def rename(self, name):
     def get(self, fields=None, etag=None):
         """Base class override.
 
+        :param fields:
```

Comment:
This is fine to leave as-is. Though the reason it wasn't documented before is that we usually don't re-document params for baseclass overrides.